pxr/usd/usdGeom/schema.usda

```diff
@@ -2292,6 +2292,35 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
+    These parameters are provided by \\ref UsdGeomCamera::GetExposureTimeAttr(),
+    \\ref UsdGeomCamera::GetExposureIsoAttr(), \\ref UsdGeomCamera::GetExposureFNumberAttr(), 
+    and \\ref UsdGeomCamera::GetExposureAttr(), respectively. 
+    \\ref UsdGeomCamera::GetExposureResponsivityAttr() provides an additional scaling
+    factor to model the overall responsivity of the system, including response of 
+    the sensor and loss by the lens.
+
+    The combination of these parameters gives an exposure scaling factor (also 
+    referred to as the __imaging ratio__) that converts from incident luminance 
+    passing through the lens system and reaching the sensor, in __nits__, to an 
+    output signal based on photometric exposure in __lux-seconds__. 
```

Comment:
For clarity, is the "imaging ratio" the factor that converts the luminance in nits=cd/m^2 (=lux/radians^2, I think) into exposure in lux-seconds (so [per-wavelength perceptually adjusted] energy per area) of the film? -So "imaging ratio" has unit radians^2 second(??)? Or is this the factor that converts it all the way from luminance to actual the unit-less floating point value that I want to write into some image file, e.g., OpenEXR? So it's unit is simply 1/nits?

Since the factor also includes the iso, it seems to me that it is the latter. In that case, I guess exposureResponsivity should have unit "1/(nits * 100ISO)" - if ISO were an actual unit.

Minor nitpick: "passing through" sounds like you only look at the light that makes it through, say, the coating of a lens. Would "Enter the lens system" be more correct? And where would you author what fraction of light doesn't make it through the lens system?

Note: It might be nice to remind the reader that nits = cd/m^2 in SI units. Bonus: lux and nits should actually be related by radians^2. Or even better, maybe say that ignoring units, the "imaging ratio" is the value of the output signal you get if you had nits luminance entering the lens or equivalently a large Lambertian radiator far from the lens radiating with f lux (where f should be 1/area of sphere, I think, I might be missing some factor here).

Reply:
> Minor nitpick: "passing through" sounds like you only look at the light that makes it through, say, the coating of a lens. Would "Enter the lens system" be more correct? And where would you author what fraction of light doesn't make it through the lens system?

This is the light that gets all the way through to the sensor. The fraction of light that doesn't make it (due to being absorbed) is 1-responsivity.

Reply:
Thanks.

Reading "image ratio", I want to know what the numerator and the denominator is. Currently, it says "factor that converts from [1] to [2]". So I should be able to read of the denominator from [1] and numerator from [2]. [1] ends in "reaching the sensor". So if I read this literally, "image ratio" does not account for anything that happens in front of the sensor such as light being absorbed by lens coating. But you are just saying that such an affect would be accounted for by the responsivity.

I know that an expert would probably not read it this way. But I think the wording in the usd schema should be unambiguous. Breaking up that sentence should achieve this.

Reply:
I'm sorry I'm not following, when you say "break up the sentence" what do you mean exactly?

Reply:
I was thinking of first defining the "imaging ratio" by unambiguously saying what it is the ratio of or what it is converting from and to. And then listing intermediate steps or contributing factors.

"from incident luminance passing through the lens system and reaching the sensor" can be interpreted in multiple ways.

Reply:
OK I've added the units of the imaging ratio (sr.s) and noted that nits are cd/m^2.

Reply:
@unhyperbolic pushed another update that fixes my bad dimensional analysis and more explicitly clarifies the conversion and units involved. Let me know what you think.

Reply:
@unhyperbolic polite bump here - does this address your concerns?